lightpath/ui/widgets.py

```diff
@@ -18,12 +18,15 @@
 logger = logging.getLogger(__name__)
 
 # Define the state colors that correspond to DeviceState
-state_colors = [QColor(124, 252, 0),  # Removed
-                QColor(255, 0, 0),  # Inserted
-                QColor(255, 215, 0),  # Unknown
-                QColor(255, 215, 0),  # Disconnected
-                QColor(255, 0, 255),  # Disconnected
-                QColor(255, 0, 255)]  # Error
+state_colors = {
+    'removed': QColor(124, 252, 0),  # Removed (green)
+    'half_removed': QColor(0, 176, 255),  # half-removed (light blue)
```

Comment:
Is this the "inserted but not blocking" state?

Reply:
Yep, I'm sure there's a better name but that's what I went with for now.

Reply:
I also have no better suggestion despite it seeming like one should exist